de.dlr.sc.virsat.graphiti.ui/src/de/dlr/sc/virsat/graphiti/ui/diagram/editor/VirSatDiagramUpdateBehavior.java

```diff
@@ -146,8 +150,14 @@ protected void createEditingDomain(IDiagramEditorInput input) {
 		// Dont create a new editing domain, instead get the virsat editing domain
 		// associated with the project of this file 
 		DiagramEditorInput diagramInput = (DiagramEditorInput) input;
-		Path path = new Path(diagramInput.getUri().toPlatformString(false));
-		IResource resource = ResourcesPlugin.getWorkspace().getRoot().findMember(path);
+		String decodedPath = null;
+		try {
+			decodedPath = URLDecoder.decode(diagramInput.getUri().toPlatformString(false), StandardCharsets.UTF_8.name());
+		} catch (UnsupportedEncodingException e) {
+			e.printStackTrace();

```

Comment:
Please report errors via the logging service

Reply:
Added the Activator for logging service